libopenage/assetmanager.cpp

```diff
@@ -17,10 +17,11 @@
 
 namespace openage {
 
-AssetManager::AssetManager(util::Dir *root)
+AssetManager::AssetManager(gui::GuiItemLink *gui_link)
 	:
-	root{root},
-	missing_tex{nullptr} {
+	root{std::string()},
```

Comment:
You can leave out this init. An error should occur if the data dir wasn't set but is fetched, otherwise funny bugs may occur?